src/options/Keybindings/default.json

```diff
@@ -402,6 +411,7 @@
         "openLinkInBackgroundTab": [
           [{ "code": "KeyF", "key": "F", "shiftKey": true }]
         ],
+        "openMultipleLinksInBackgroundTabs": [],
```

Comment:
I was thinking of a shortcut in the form of ALT+something but, at least in linux, they open the browser's menus like File, Edit, and stuff like that. Let me know if you want some specific defaults.